hugegraph-core/src/main/java/com/baidu/hugegraph/backend/tx/SchemaTransaction.java

```diff
@@ -70,12 +72,16 @@
 public class SchemaTransaction extends IndexableTransaction {
 
     private final SchemaIndexTransaction indexTx;
+    private final SystemSchemaStore systemSchemaStore;
+    private final LocalCounter counter;
```

Comment:
TODO: move `LocalCounter counter` define into SystemSchemaStore class

Reply:
LocalCounter provider by HugegraphParams, if we move it to SystemSchemaStore, maybe change too much code